src/components/notifications.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import Badge from '@material-ui/core/Badge';
+import Popper from '@material-ui/core/Popper';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import Grow from '@material-ui/core/Grow';
+import messages from '../notifications.json';
+import orderBy from 'lodash/orderBy';
+import filter from 'lodash/filter';
+import { NotificationList } from './notification-list';
+
+const isBrowser = typeof window !== 'undefined';
+const storageItem = isBrowser
+  ? localStorage.getItem('lastSeenNotification')
+  : null;
+const lastSeenNotification = storageItem ?? '2000-1-1';
+const messageList = orderBy(messages, ['date'], ['desc']);
```

Comment:
I don't think the args need to be arrays.
```suggestion
const messageList = orderBy(messages, 'date', 'desc');
```